README.md

```diff
@@ -43,7 +39,7 @@ import ReactHighcharts from 'react-highcharts';
 ```
 This will use the default sample data to render a chart in your application. Read the following section for instructions on specifying your own configuration/data.
 
-For more detailed instructions on using HighCharts in your application, see our demos for [Angular](https://stackblitz.com/edit/pxblue-highcharts-angular) or [React](https://stackblitz.com/edit/pxblue-highcharts-react).
+For more detailed instructions on using HighCharts in your application, see our demos for [Angular](https://stackblitz.com/github/pxblue/highcharts/tree/master/angular-demo) or [React](https://codesandbox.io/s/github/pxblue/highcharts/tree/master/react-demo).
```

Comment:
Let's stick with the table version of the demos at the end (see themes or chartjs) since they don't really add any more explanation. It's a little weird having this divide basic from advanced usage in this place.